root/Makefile

```diff
@@ -199,6 +199,7 @@ docs-publish:: pre-docs-publish
 ## fly-login:       log into concourse
 .PHONY: fly-login
 fly-login:
+	ls ${FLY} 2>/dev/null || curl -s 'https://concourse.outreach.cloud/api/v1/cli?arch=${GOARCH}&platform=${GOOS}' -o ${FLY} && chmod +x ${FLY} && ${FLY} --version
```

Comment:
This is Outreach specific, so we can't put this in devbase. I'd suggest using a system version of fly, e.g. `brew install fly` and then doing the `fly sync` logic here instead.

Reply:
I just noticed we have the address in here on L202 as well, so we should fix that at the same time 😭 

We could probably put the address in the box config to make it easier as well

Reply:
just updated with brew and it works